test/spec.index.js

```diff
@@ -973,6 +973,12 @@ describe('Template Mixins', function () {
                 res.locals.url().call(res.locals, 'path').should.equal('/base/path');
             });
 
+            it('returns path if baseUrl is not set', function () {
+                req.baseUrl = undefined;
+                middleware(req, res, next);
+                res.locals.url().call(res.locals, 'path').should.equal('path');
```

Comment:
Could you also add a check for './path' please. Otherwise, this looks good to me.